scripts/dts/extract/globals.py

```diff
@@ -25,7 +25,8 @@
 regs_config = {
     'zephyr,sram'  : 'DT_SRAM',
     'zephyr,ccm'   : 'DT_CCM',
-    'zephyr,dtcm'  : 'DT_DTCM'
+    'zephyr,dtcm'  : 'DT_DTCM',
+    'zephyr,ipc_shm' : 'DT_IPC_SHM'
```

Comment:
Expect conflicts with #17660

Reply:
#17660 is now merged, you'll have to report this patch in `scripts/dts/gen_defines.py`

Reply:
done